Booktrix/Controllers/SignStoryboard/SignInViewController.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  SignInViewController.swift
+//  Booktrix
+//
+//  Created by Impresyjna on 14.05.2017.
+//  Copyright © 2017 Impresyjna. All rights reserved.
+//
+
+import UIKit
+
+class SignInViewController: UIViewController, UITextViewDelegate {
+    @IBOutlet weak var loginTextField: UITextField!
+    @IBOutlet weak var passwordTextField: UITextField!
+    
+    var viewModel = SignInViewModel()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        self.hideKeyboardWhenTappedAround()
+        prepareFields()
+    }
+    
+    func prepareFields() {
+        loginTextField.prepareForView()
+        passwordTextField.prepareForView()
+    }
+    
+    @IBAction func textFieldChanged(_ sender: Any) {
+        viewModel.form.login = loginTextField.text ?? ""
+        viewModel.form.password = passwordTextField.text ?? ""
+    }
+    
+    //MARK: Button actions
+    @IBAction func openSignUp(_ sender: Any) {
+        self.presentViewFromStoryboard(storyboardName: "SignUpView", viewName: "SignUpViewController")
```

Comment:
pozbył bym się tych stringów. To nie jest dobry pomysł polegać na nich bezpośrednio. Nazwy storyboardów możesz ogarnąć robiąc sobie `enum StoryboardName: String` w jakimś pliku i w case'ach wypisywać jakie masz storyboardy. Co to `viewName`, zrób extension na UIViewController który zwraca `static var identifier: { String(describing: self) }` to automatycznie robi stringa z nazwy klasy kontrolera. Zobacz jak mam ogarnięte `Wireframe` w Just. Jest dużo czyściej.